docker/images.json

```diff
@@ -1,15 +1,24 @@
 {
   "images": {
     "m3dbnode": {
+      "name": "m3dbnode",
       "dockerfile": "docker/m3dbnode/Dockerfile"
     },
+    "m3dbnode-setcap": {
+      "name": "m3dbnode",
+      "dockerfile": "docker/m3dbnode/Dockerfile",
```

Comment:
Hm don't we need a new dockerfile that actually calls setcap?

Reply:
🤦‍♂ will add